src/test/test_interval_bitwise.cpp

```
@@ -30,6 +30,25 @@ TEST_CASE("signed division by a negative singleton preserves interval order", "[
+    // dividends, letting the verifier prune reachable paths.
+
+    // -8 s/ 5 = -1 and -8 s/ 6 = -1  (both truncate toward zero to -1).
+    REQUIRE(Interval{-8, -8}.sdiv(Interval{5, 6}) == Interval{-1, -1});
+    REQUIRE((Interval{-8, -8} / Interval{5, 6}) == Interval{-1, -1});
+
+    // -7 s/ 2 = -3, -7 s/ 3 = -2, -5 s/ 2 = -2, -5 s/ 3 = -1  ->  [-3, -1].
+    REQUIRE(Interval{-7, -5}.sdiv(Interval{2, 3}) == Interval{-3, -1});
+
+    // Positive and negative/negative boxes were already exact and stay so.
+    REQUIRE(Interval{7, 8}.sdiv(Interval{2, 3}) == Interval{2, 4});
+    REQUIRE(Interval{-8, -6}.sdiv(Interval{-3, -2}) == Interval{2, 4});
```

Cppcheck (2.21.0)

[performance] Function parameter 'variable_terms' should be passed by const reference.

(passedByValue)